ui/apps/dashboard/src/components/Navigation/Logo.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
+
+import type { Route } from 'next';
+import Link from 'next/link';
+import { RiContractLeftLine, RiContractRightLine } from '@remixicon/react';
+
+import { toggleNav } from '@/app/actions';
+import InngestLogo from '@/icons/InngestLogo';
+import { InngestLogoSmall } from '@/icons/InngestLogoSmall';
+import Search from './Search';
+
+type LogoProps = {
+  collapsed: boolean;
+  setCollapsed: (arg: boolean) => void;
+};
+
+const NavToggle = ({ collapsed, setCollapsed }: LogoProps) => {
+  const toggle = () => {
+    setCollapsed(!collapsed);
+    toggleNav();
+  };
+
+  return collapsed ? (
+    <RiContractRightLine
+      className="bg-canvasBase text-subtle invisible h-5 w-5 cursor-pointer group-hover:visible"
+      onClick={toggle}
+    />
+  ) : (
+    <RiContractLeftLine
+      className="bg-canvasBase text-subtle invisible h-5 w-5 cursor-pointer group-hover:visible"
+      onClick={toggle}
+    />
+  );
+};
+
+export default function Logo({ collapsed, setCollapsed }: LogoProps) {
+  return (
+    <div className="group ml-5 mr-4 mt-5 flex h-10 flex-row items-center justify-between">
+      <div className="flex flex-row items-center justify-start">
+        {collapsed ? (
+          <div className="cursor-pointer transition-all delay-150 duration-300 group-hover:hidden">
+            <InngestLogoSmall />
```

Comment:
Small logo is not yet color scheme away. I couldn't get that working with the underlying svg quickly. I'll ticket and tackle before release.

Reply:
Might be able to use fill=currentColor in the svg and then use a text color on the parent element to flip it along side the them

Reply:
Yeah, the logo is two colors that will have to flip based on color scheme so I'll have to sort that out. Will round back and do that soon.